gemmbench/problems.py

```diff
@@ -885,16 +873,27 @@ def get_gemm_configs() -> list[tuple[str, GemmConfig]]:
     configs += [("llama70bmemory", x) for x in llama70bmemory_configs]
     configs += [("compute", x) for x in compute_configs]
     configs += [("unet", x) for x in unet_configs]
-    configs += [("tk", x) for x in tk_default_configs]
     
     return configs
 
 def get_tk_gemm_configs() -> list[tuple[str, GemmConfig]]:
     configs: list[tuple[str, GemmConfig]] = []
-    tk_default_configs = tk_default("f16")
-    tk_unet_configs = tk_unet("f16")
+    llama13bmatvec_configs = llama13bmatvec("f16")
+    llama70bmatvec_configs = llama70bmatvec("f16")
+    llama13bskinny_configs = llama13bskinny("f16")
+    llama70bskinny_configs = llama70bskinny("f16")
```

Comment:
This will add all the transpose A and no transpose matmuls to the tk testing as well. Pretty sure it will result in a failure

Reply:
Thanks, I will remove those configs.